app/models/work_package.rb

```diff
@@ -138,7 +138,7 @@ class WorkPackage < ApplicationRecord
          :relatable,
          :directly_related
 
-  acts_as_watchable
+  acts_as_watchable(permission: :view_work_packages)
```

Comment:
We have classes that are derived from the `WorkPackage` class like `Task` or `Impediment`. When working with those classes, the permission check used to auto-generate the permission name that was checked. This generated `view_tasks` and `view_impediments`, which don't exist. Specifying the permission prevents the auto generation and always uses the `view_work_packages` permission correctly.